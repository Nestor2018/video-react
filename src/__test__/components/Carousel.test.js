import React from 'react';
import { mount } from 'enzyme';
import { create } from 'react-test-renderer';

import Carousel from '../../components/Carousel';

describe('<Carousel/>', () => {
	const carousel = mount(<Carousel />);
	test('Render del componente Carousel', () => {
		expect(carousel.length).toEqual(1);
	});
});

describe('Carousel Snapshot', () => {
	const carousel = create(<Carousel />);
	test('Comprobar la Ui del componente Carouse', () => {
		expect(carousel.toJSON()).toMatchSnapshot();
	});
});
