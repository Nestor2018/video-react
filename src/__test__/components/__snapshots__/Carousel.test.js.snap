// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel Snapshot Comprobar la Ui del componente Carouse 1`] = `
<section
  className="carousel"
>
  <div
    className="carousel__container"
  />
</section>
`;
